src/coreclr/tools/Common/TypeSystem/Common/TargetDetails.cs

```diff
@@ -185,6 +185,8 @@ public int MinimumCodeAlignment
             }
         }
 
+        public int MinimumObjectSize => PointerSize * 3;
```

Comment:
This GC-specific detail. I do not think that this belongs to TargetDetails. 

EETypeNode looks like the closest home. I would make it a static method on `EETypeNode`, something like:

```
static public int GetMinimumObjectSize(NodeFactory factory)
    => factory.TypeSystemContext.Target.PointerSize * 3;
```